crates/modelardb_common/src/arguments.rs

```diff
@@ -106,63 +106,26 @@ pub fn argument_to_connection_info(argument: &str) -> Result<Vec<u8>, String> {
     }
 }
 
-/// The object stores that are currently supported as remote data folders.
-#[derive(PartialEq, Eq)]
-pub enum RemoteDataFolderType {
-    S3,
-    AzureBlobStorage,
-}
-
-impl FromStr for RemoteDataFolderType {
-    type Err = String;
-
-    fn from_str(value: &str) -> Result<Self, Self::Err> {
-        match value {
-            "s3" => Ok(RemoteDataFolderType::S3),
-            "azureblobstorage" => Ok(RemoteDataFolderType::AzureBlobStorage),
-            _ => Err(format!(
-                "'{}' is not a valid value for RemoteDataFolderType.",
-                value
-            )),
-        }
-    }
-}
-
-/// Extract the remote data folder type from the arguments and validate that the remote data folder can be
-/// accessed. If the remote data folder cannot be accessed, return the error that occurred as a [`String`].
-pub async fn validate_remote_data_folder_from_argument(
-    argument: &str,
-    remote_data_folder: &Arc<dyn ObjectStore>,
-) -> Result<(), String> {
-    if let Some(split_argument) = argument.split_once("://") {
-        let object_store_type = split_argument.0;
-        let remote_data_folder_type = RemoteDataFolderType::from_str(object_store_type)?;
-
-        validate_remote_data_folder(remote_data_folder_type, remote_data_folder).await
-    } else {
-        Err(format!(
-            "Remote data folder argument '{argument}' is invalid."
-        ))
-    }
-}
-
 /// Validate that the remote data folder can be accessed. If the remote data folder cannot be
 /// accessed, return the error that occurred as a [`String`].
 pub async fn validate_remote_data_folder(
-    remote_data_folder_type: RemoteDataFolderType,
     remote_data_folder: &Arc<dyn ObjectStore>,
 ) -> Result<(), String> {
+    let invalid_path = Uuid::new_v4().to_string();
```

Comment:
Maybe add a comment why a UUID is used as `invalid_path` instead of a pre-generated random path so it is clear if we need to change the code at a later date, I assume it is to minimize the change of using a filename that exists.

Reply:
Fixed.